src/servicetools/logging_config.py

```diff
@@ -37,6 +38,20 @@ def level(self) -> int:
         return VERBOSE_LEVELS[v] if v < len(VERBOSE_LEVELS) else VERBOSE_LEVELS[-1]
 
 
+def _log_uncaught_exceptions(
+    exception_class: type[BaseException],
```

Comment:
In python versions before 3.9, i think its best to use `Type` instead of `type`.